src/test/mock/CustomAxiosMockAdapter.ts

```diff
@@ -0,0 +1,429 @@
+import { defaultUrl } from '@googlemaps/google-maps-services-js/dist/places/details';
+import MockAdapter from 'axios-mock-adapter';
+import { Status } from '@googlemaps/google-maps-services-js';
+import { restaurantDetailsFields } from '../../main/service/GooglePlaceService';
+import { PlaceField } from '../../main/data/PlaceField';
+
+/**
+ * Wrapper class around MockAdapter to set custom mocking behaviour
+ * * To mock axios requests, add a new function and implement the expected response. Then in your test, call the function
+ * that was implemented to configure the mock for the respective axios instance.
+ * */
+export class CustomAxiosMockAdapter extends MockAdapter {
```

Comment:
We are using [axios-mock-adapter](https://www.npmjs.com/package/axios-mock-adapter) to mock [google-maps-service-js](https://googlemaps.github.io/google-maps-services-js/index.html). 

There's is no mocking library for google-maps-service. But the good thing about google-maps-service is that its constructor takes in an `axios` instance. So we can mock the `axios` instance which is realy neat!